models/attribute-group/src/transformers.ts

```diff
@@ -0,0 +1,24 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TAttributeGroup, TAttributeGroupGraphql } from './types';
+const transformers = {
+  graphql: Transformer<TAttributeGroup, TAttributeGroupGraphql>('graphql', {
```

Comment:
Don't we need the rest transformer so the `createdBy`, `lastModifiedBy`, `name` and `description` properties are actually built for that representation?

Reply:
When we (Pacman) worked on the attribute group, we decided to consume only the GraphQL in FE. I didn't see any benefit in creating rest since it is not used. Do you think for the consistency, every model should have both ?

Reply:
Done in https://github.com/commercetools/test-data/pull/652/commits/ac5cfe5c389dddd194f3880105d4b470b2da6899

Reply:
Thanks for addressing this comment.

Yes, I think we should keep the consistency in all models. Also because this is a public repo and other consumers might need to use the rest representation.